charts/fission-all/templates/buildermgr/deployment.yaml

```diff
@@ -14,6 +14,10 @@ spec:
     metadata:
       labels:
         svc: buildermgr
+      annotations:
+        prometheus.io/scrape: "true"
+        prometheus.io/path: "/metrics"
+        prometheus.io/port: "8080"
     spec:
       containers:
       - name: buildermgr
```

Comment:
We need to expose metrics port also. Same changes required bellow in controller and storagesvc.